src/main/kotlin/ink/abb/pogo/scraper/util/pokemon/CatchablePokemon.kt

```diff
@@ -48,7 +49,7 @@ fun CatchablePokemon.catch(captureProbability: CaptureProbability, itemBag: Item
         }
         if(waitBetweenThrows){
             val waitTime = (Math.random()*2900 + 100)
-            Log.blue("Missed, waiting for ${waitTime/1000} seconds")
+            Log.blue("Damn, the Pokemon got out of the ball. Waiting for ${DecimalFormat("#0.0").format(waitTime/1000)} seconds while aiming the next ball")
```

Comment:
Something like remove  "Damn, the " and "while aiming the next ball" from the log string